src/store/actions.ts

```diff
@@ -333,7 +333,7 @@ export const fetchMRASProfile = async ({ commit, getters }): Promise<any> => {
         const msg = await NamexServices.handleApiError(err, 'Could not fetch mras profile')
         console.error('fetchMRASProfile() =', msg) // eslint-disable-line no-console
       }
-      commit('mutateName')
```

Comment:
This caused `name` to be undefined, which caused some de-reference error (eg, "cannot call replace on undefined").